projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss

```diff
@@ -40,3 +40,26 @@
 	.jetpack-upgrade-plan-banner.wp-block[data-align=right] .jetpack-upgrade-plan-banner__wrapper, .jetpack-upgrade-plan-banner.wp-block[data-align=left] .jetpack-upgrade-plan-banner__wrapper {
 		max-width: 580px;
 		width: 100%; }
+
+
+		
+// Styling copied from Gutenberg's `@wordpress/block-editor` styling for the Warning component
```

Comment:
The first line of the style.scss file that we're removing is `@import '../styles/gutenberg-base-styles.scss';` Does this need to include the `gutenberg-base-styles.scss` file once we've moved it to the base styles package?

Reply:
Yeah, we should do that. I will get it done once [this](https://github.com/Automattic/jetpack/pull/23817) gets merged